docs/standard/translation.md

```diff
@@ -22,12 +22,15 @@ We currently actively maintain translations to:
 
 A number of other community translations exist in various levels of completion.
 
-## Translators and reviewers
+## Translators, proofreaders and reviewers
 
-```eval_rst
-  .. todo::
-    Describe the process for selecting translators and reviewers and managing permissions.
-```
+Translators, proofreaders and reviewers have excellent writing skills (spelling and grammar) and intervene only when the target language is their native language.
+
+**Translators** are the persons who write the source content in one of the target languages. They must be familiar with the topics (public contracting and data) and apply the guidelines (this document) and terminology (in Transifex) provided by the Open Contracting Partnership.
```

Comment:
Remove "are the persons who"

Reply:
Translation.md deals only with translation, and the terminologist is only involved in the terminology process. I consequently don't think we should mention this role here.